testcases/OpTestLPM.py

```diff
@@ -236,6 +243,27 @@ def rmc_service_start(self, mg_system, remote_hmc=None, output_dir=None):
                 self.collect_logs_test_fail(remote_hmc, output_dir)
                 raise OpTestError("RMC connection is down!!")
 
+    def execute_stressng(self):
+        thread_timeout = re.search('(?:--timeout|-t) (\d+[a-zA-Z]?)', self.stressng_command)
+        if not thread_timeout:
+            self.stressng_command = self.stressng_command + ' --timeout 10m'
+            thread_timeout = 10
+        else:
+            conversion_dict = {'s':1/60, 'm':1, 'h':60, 'd':1440, 'y':525600}
+            thread_timeout = thread_timeout.group(1)
+            if thread_timeout[-1].isdigit():
+                thread_timeout = int(thread_timeout)*conversion_dict['s']
+            elif thread_timeout[-1].lower() in conversion_dict:
+                thread_timeout = int(thread_timeout[:-1])*conversion_dict[thread_timeout[-1].lower()]
```

Comment:
ah i just lost here  , can you please explain me what time out value we are looking

Reply:
When creating OpSSHThreadLinearVar1() object, we need to provide an execution_time parameter (which is supposed to be in minutes).
So, this just takes care of converting the stress-ng timeout to minutes, in order to pass it as an argument to OpSSHThreadLinearVar1()

Reply:
@PraveenPenguin we did had a live debug session on stress-ng,  basically for a given lpar .. with the given resource... it is user responsibility to tune the stress-ng command as per lpar resources.. and we have seen with 30 secs the lpar is loaded with all resources being used by stress-ng wload,,.. so  by default we give 10 mins  of stress-ng  .. but user can input 30mins or 1hr from config file.. and for 30 mins .. the every new stress-ng thread is created every 10 mins... so basically every 10 mins all resources are relaoded back .. also make sure no OOM killer is being called so test should not stuck